frontend_tests/node_tests/pm_conversations.js

```diff
@@ -29,20 +30,29 @@ run_test("partners", () => {
 run_test("insert_recent_private_message", () => {
     const params = {
         recent_private_conversations: [
-            {user_ids: [11, 2], max_message_id: 150},
+            {user_ids: [11, 2, 8], max_message_id: 150},
             {user_ids: [1], max_message_id: 111},
+            {user_ids: [3], max_message_id: 211},
             {user_ids: [], max_message_id: 7},
         ],
     };
     people.initialize_current_user(15);
     initialize_recents(params);
 
+    muting.add_muted_user(2);
+    muting.add_muted_user(3);
+
     assert.deepEqual(pmc.recent.get(), [
-        {user_ids_string: "2,11", max_message_id: 150},
+        {user_ids_string: "2,8,11", max_message_id: 150},
         {user_ids_string: "1", max_message_id: 111},
         {user_ids_string: "15", max_message_id: 7},
     ]);
+    assert.deepEqual(pmc.recent.get_strings(), ["2,8,11", "1", "15"]);
```

Comment:
Can you do a `get_strings` check just before muting these users, just so the test makes the change really clear?

Reply:
Done, I separated this out into its own test, along with some prep cleanup.